uaparser/user_agent.go

```diff
@@ -14,6 +14,7 @@ type UserAgent struct {
 type UserAgentPattern struct {
 	Regexp            *regexp.Regexp
 	Regex             string
+	RegexFlag		  string
```

Comment:
Yeah, was just lazy. Doesn't look like there's any pre-commit hooks in this project to ensure go fmt is run. Probably should have that.